### PR TITLE
use get_content for SC RSS

### PR DESCRIPTION
**Pull request by MichaelButkovic**

### CODE REVIEW

**Comment by MichaelButkovic** [comment]
@mparent61 @benghaziboy

**Comment by benghaziboy** [comment]
LGTM, worth noting that percolator tests are a giant pain right now. I'll have to shore that up.

File: bulbs/feeds/views.py [hunk 1]
```diff
@@ -54,10 +54,10 @@ def get_queryset(self):
...
 
         if sc_id:
-            sc_ids = SpecialCoverage.objects.get(id=sc_id).query["included_ids"]
+            sc = SpecialCoverage.objects.get(id=sc_id)
         elif sc_slug:
-            sc_ids = SpecialCoverage.objects.get(slug=sc_slug).query["included_ids"]
+            sc = SpecialCoverage.objects.get(slug=sc_slug)
         else:
-            sc_ids = []
+            return self.model.objects.none()
```
Comment (mparent61) [inline bulbs/feeds/views.py]: Didn't know about this one!

**Comment by mparent61** [comment]
:+1:

**Pull request merged**